packages/terra-theme-provider/src/ThemeProvider.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ThemeProviderThemes = {
+  CONSUMER: 'cerner-consumer-theme',
+  MOCK: 'cerner-mock-theme',
+};
+
+const propTypes = {
+  /**
+   * The component(s) that will be wrapped by `<ThemeProvider />`
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Name of class for specified theme. e.g `cerner-consumer-theme`
+   */
+  themeName: PropTypes.string,
+};
+
+const ThemeProvider = ({ themeName, children, ...customProps }) => (
+  <div {...customProps} className={themeName}>{children}</div>
+);
+
+ThemeProvider.propTypes = propTypes;
+ThemeProvider.Opts = {};
+ThemeProvider.Opts.Themes = ThemeProviderThemes;
```

Comment:
can user pass a theme name which is not included in the opts? Is it just a predefined mapping so that user can use it like: 
`<ThemeProvider themeName={ThemeProvider.Opts.Themes.MOCK}>`
 instead of 
`<ThemeProvider themeName="terra-mock-theme"}>`?